cmd/homi/setup/flags.go

```diff
@@ -53,14 +53,14 @@ var (
 	}
 
 	numOfCNsFlag = cli.IntFlag{
-		Name:  "num",
+		Name:  "cn-num",
 		Usage: "Number of consensus nodes",
 		Value: 4,
 	}
 
 	numOfValidatorsFlag = cli.IntFlag{
 		Name:  "num-validators",
-		Usage: "Number of validators. If not set, it will be set the number of option num",
+		Usage: "Number of validators. If not set, it will be set the number of option cn-num",
 		Value: 0,
```

Comment:
Maybe you can declare a const `const DefaultNumCNs = 4` and used this both for `numOfCNsFlag` and `numOfValidatorsFlag`. Since displaying 0 as default value might confuse the users.

Reply:
Below code prevents the case. :) So it's ok.
github.com/klaytn/klaytn/cmd/homi/setup/cmd.go:426
```
	if numValidators > num {
		return fmt.Errorf("num-validators(%d) cannot be greater than num(%d)", numValidators, num)
	}
```